scipy-1.0/paper.tex

```diff
@@ -28,7 +28,19 @@
 \keywords{Scientific computing, Python, Mathematics}
 
 \begin{abstract}
-TODO. Abstract must be under 200 words and not include subheadings or citations.
+SciPy is an open source scientific computing library for the Python programming language.
+SciPy 1.0 was released in late 2017, about 16 years after the original
+version 0.1 release. SciPy
+has become a \emph{de facto} standard in the community, with more than 600
```

Comment:
Given the broad scope of the journal, a reader might not understand which community this refers to - the Python community, or the scientific (computing) community? (Of course, the term SciPy also refers to the [SciPy ecosystem ](https://www.scipy.org/) as a whole (including matplotlib, sympy, pandas, etc...), so it could conceivably mean that community, too.) On the other hand, if you were to state explicitly "a _de facto_ standard for scientific computing in the Python community", this would seem redundant. So maybe it's fine as-is, but perhaps consider how it might be more explicit.